sdk/Trace/SpanProcessor/BatchSpanProcessor.php

```diff
@@ -126,7 +126,14 @@ protected function queueReachedLimit(): bool
 
     protected function enoughTimeHasPassed(): bool
     {
-        $now = $this->clock->timestamp();
+        $now = (int) ($this->clock->now() / 1e6);
```

Comment:
We could use the [RealtimeClock](https://github.com/open-telemetry/opentelemetry-php/blob/master/sdk/Trace/RealtimeClock.php) Class to handle this, but not a blocker.

Reply:
"Has time passed?" is a duty of a monotonic clock. If it relates to time in the real world "What time is it right now?" and such then it's a concern for a Realtime clock.